service_container/service_subscribers_locators.rst

```diff
@@ -859,7 +859,13 @@ the following order:
 Service Subscriber Trait
 ------------------------
 
-The :class:`Symfony\\Contracts\\Service\\ServiceSubscriberTrait` provides an
+.. deprecated:: Contracts 3.5
```

Comment:
I haven't found any previous examples of deprecation in symfony/contracts.

Reply:
You have to use `.. deprecated:: 3.5` and if the linter complains, add `- '.. deprecated:: 3.5' # Contracts` in https://github.com/symfony/symfony-docs/blob/b5f8a39bf02914b82ae2e6e119f06066f2c9afc3/.doctor-rst.yaml#L98

Reply:
That's fine because those who use the deprecated trait will have the notice with their static analysis.
Added a [comment on the PR](https://github.com/symfony/symfony/pull/54496#discussion_r1559060497) because the deprecation will appear in the logs.